shared/guide/system/software/library/package_uuidd_installed.rule

```diff
@@ -0,0 +1,23 @@
+documentation_complete: true
+
+prodtype: ol7
+
+title: 'Package uuidd Installed'
+
+description: |-
+    The package <tt>uuidd</tt> is not installed on Oracle Linux 7 by default. 
+    Using the following command to install and/or update the package:
+    <pre>$ sudo yum install uuidd</pre>
+
+rationale: |-
+    The <tt>uuidd</tt> package contains a userspace daemon (uuidd) which is 
+    used to generate unique identifiers even at very high rates on 
+    SMP systems.
+
+severity: high
```

Comment:
High findings correlate to CAT I/critical. Can justification of a the severity rating be provided?

This could be:
- Point to some standard that calls it high, e.g. the SAP security guide, DISA STIG, CIS, etc
- Write a few lines. Absolutely should not be a book. Having documented reasoning in the PR for future reference is the ask :)